package/network/services/umdns/files/umdns.init

```diff
@@ -29,6 +34,12 @@ reload_service() {
 }
 
 start_service() {
+	local enable="$(uci -q get umdns.@umdns[-1].enable)"
+
+	if [ "${enable}" == "0" ] || [ "${enable}" == "false" ]; then
```

Comment:
would make more sense to use disabled 1 and check for that.

Reply:
I believe the preferred paradigm is "enabled". The adjectival form.

Reply:
@systemcrash problem is that any old config carried over from sysupgrade won't include that option, thus not starting the service anymore

Reply:
@systemcrash and because of said reason it should be "disabled". and if that is missing the service will start. this is how the wireless config does it.

Reply:
Point taken. A service can still start if enabled is missing. Just that (script can interpret) enabled '0' would disable it.

Reply:
Updated, kindly review.